extensions/hacker-news/src/StoryListItem.tsx

```diff
@@ -1,17 +1,31 @@
-import { ActionPanel, List, Action } from "@raycast/api";
+import { ActionPanel, List, Action, Icon } from "@raycast/api";
 import Parser from "rss-parser";
 
 export function StoryListItem(props: { item: Parser.Item; index: number }) {
   const icon = getIcon(props.index + 1);
   const points = getPoints(props.item);
   const comments = getComments(props.item);
 
+  const hasComments = typeof comments !== "undefined";
+  const hasPoints = typeof points !== "undefined";
+  const getAccessoryTitle = () => {
+    const parts = [];
+    if (hasComments) {
+      parts.push(`${comments}`);
+    }
+    if (hasPoints) {
+      parts.push(`👍 ${points}`);
+    }
+    return parts.join("    ");
+  };
+
   return (
     <List.Item
       icon={icon}
       title={props.item.title ?? "No title"}
       subtitle={props.item.creator}
-      accessoryTitle={`👍  ${points}    💬  ${comments}`}
+      accessoryIcon={hasComments ? Icon.Bubble : undefined}
+      accessoryTitle={getAccessoryTitle()}
```

Comment:
Should we stick with just emojis until we have more flexible accessories ;-)

Reply:
wanted to be closer to the native extension 😛 but i'll change this back 😉

Reply:
done!